src/app/components/SocialLinks/index.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { ToggleContextProvider } from '#app/contexts/ToggleContext';
+import { ServiceContextProvider } from '#app/contexts/ServiceContext';
+import ThemeProvider from '#app/components/ThemeProvider';
+import { Services } from '#app/models/types/global';
+import socialLinksFixture from '#data/kyrgyz/topics/cvpv9djp9qqt.json';
+import {
+  Curation,
+  Summary,
+  VISUAL_STYLE,
+  VISUAL_PROMINENCE,
+} from '#app/models/types/curationData';
+import SocialLinks from '.';
+import metadata from './metadata.json';
+import readme from './README.md';
+
+const getSocialLinksData = (numberOfItems?: number): Curation => {
+  const socialLinksCuration = socialLinksFixture.data.curations.find(
+    curation => {
+      return (
+        curation.visualStyle === VISUAL_STYLE.LINKS &&
+        curation.visualProminence === VISUAL_PROMINENCE.NORMAL
+      );
+    },
+  ) as Curation;
+
+  if (!socialLinksCuration.summaries || !numberOfItems)
+    return socialLinksCuration;
+
+  return {
+    ...socialLinksCuration,
+    summaries: socialLinksCuration.summaries.slice(0, numberOfItems),
+  };
+};
+
+const Component = ({
+  curations,
+  service,
+}: {
+  curations: Curation;
```

Comment:
Can this be singular instead of plural
```suggestion
  curation: Curation;
```